src/io/mouse.js

```diff
@@ -12,21 +12,13 @@ function Mouse (runtime) {
     this.runtime = runtime;
 }
 
-Mouse.prototype.postData = function(data) {
-    if (data.x) {
-        this._x = data.x - data.canvasWidth / 2;
-    }
-    if (data.y) {
-        this._y = data.y - data.canvasHeight / 2;
-    }
-    if (typeof data.isDown !== 'undefined') {
-        this._isDown = data.isDown;
-        if (this._isDown) {
-            this._activateClickHats(data.x, data.y);
-        }
-    }
-};
-
+/**
+ * Activate "event_whenthisspriteclicked" hats if needed.
+ * @param  {int} x X position to be sent to the renderer.
+ * @param  {int} y Y position to be sent to the renderer.
```

Comment:
Another JSDoc curiosity - I've seen `int` and `float` in some places, although these aren't JavaScript types as far as I know. what is our general approach?